apis/types/error.go

```diff
@@ -20,24 +20,24 @@ type Error struct {
 }
 
 // Validate validates this error
-func (m *Error) Validate(formats strfmt.Registry) error {
-	return nil
+func (m *Error) Validate(formats strfmt.Registry) (err error) {
+	return err
 }
 
 // MarshalBinary interface implementation
-func (m *Error) MarshalBinary() ([]byte, error) {
+func (m *Error) MarshalBinary() (b []byte, error) {
```

Comment:
It looks like so wired. One return parameter is anonymous and another one has a name.